src/Tribe/Timezones.php

```diff
@@ -126,45 +126,17 @@ public static function event_end_timestamp( $event_id, $timezone = null ) {
 	 *
 	 * @param int    $event_id
 	 * @param string $type (expected to be 'Start' or 'End')
-	 * @param string $timezone
 	 *
 	 * @return int
 	 */
-	protected static function get_event_timestamp( $event_id, $type = 'Start', $timezone = null ) {
+	protected static function get_event_timestamp( $event_id, $type = 'Start' ) {
```

Comment:
As discussed via Slack, I intended this as a starting point toward fixing timestamps throughout our code base.

For example, implementing this fix alone without touching anything else causes the single event page's display of the start-end times to display UTC time alongside the correct timezone abbreviation, which makes the entirety of the displayed line incorrect.